modules/secrets-store-aws.adoc

```diff
@@ -0,0 +1,370 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+//
+// IMPORTANT: This file requires you to define :secrets-store-provider: before including this module.
+
+ifeval::["{secrets-store-provider}" == "AWS Secrets Manager"]
+:aws-secrets-manager:
+endif::[]
+ifeval::["{secrets-store-provider}" == "AWS Systems Manager Parameter Store"]
+:aws-systems-manager-parameter-store:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="secrets-store-aws_{context}"]
+= Mounting secrets from {secrets-store-provider}
+
+You can use the {secrets-store-operator} to mount secrets from {secrets-store-provider} to a CSI volume in {product-title}. To mount secrets from {secrets-store-provider}, your cluster must be installed on AWS and use AWS Security Token Service (STS).
+
+[IMPORTANT]
+====
+It is not supported to use the {secrets-store-operator} with {secrets-store-provider} in a hosted control plane cluster.
+====
+
+.Prerequisites
+
+* Your cluster is installed on AWS and uses AWS Security Token Service (STS).
+* You have installed the {secrets-store-operator}. See _Installing the {secrets-store-driver}_ for instructions.
```

Comment:
```suggestion
* You have installed the {secrets-store-operator}. See "Installing the {secrets-store-driver}" for instructions.
```
I'm never sure about this one, but when referring to headings or subsections, should the name be in quotation marks and not italics? [ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-citations#general-guidelines) says: "If you refer to a smaller piece of work within a larger container such as a chapter within a book or an article in a magazine, use italic font for the larger container and double quotation marks for the smaller piece" though I don't know if this pertains to sections of a web site or not.

Reply:
@bburt-rh I agree, I've been thinking for awhile that this is something we need to standardize on. We are definitely inconsistent, at least in the OCP docs repo. 

I always do italics, so I'm going to leave this be for now, but I'd be happy to use quotes if we decide that's the way to go.

Think it's worth bringing up at the SSG level, or just within OCP docs?